docs/source/graphs.rst

```diff
@@ -0,0 +1,83 @@
+.. _reports:
+
+Graphs
+********
+
+Graph Your Data
+=================
+
+We can plot some of our data as seaborn histogram plots. Below will demonstrate how to do so and provide examples.
```

Comment:
maybe we should list the plots we have available (just 1 right now)
```
The following plots are currently available to work directly with your profilers:

 * histogram (numeric columns only)

```

Below we will demonstrate how to do so with examples.